### ISSUE TITLE
Videos on nascar.com

### ISSUE THREAD

**Issue by xenia-09A** [hagezi/dns-blocklists#2450]
Hi On nascar.com videos stopped playing when using pro ++ list and chrome , they worked before I believe
 
Pro ++ seem to be blocking ads only so it not 100% clear what blacklisted domain is breaking this 
so  can you advise if they are force breaking this and requires ad domains to be open to view these videos?

this is breaking with NExtDNS DoH and pro++ list , removing  Pro++ list for few seconds video plays

thanks
I'm a big fan thanks for all you do

**Comment by hagezi** [comment]
@xenia-09A Thanks, I'll have to see if I can reproduce that in my region. Do you perhaps have the block page in NextDNS active?

**Comment by hagezi** [comment]
@xenia-09A try unblocking: 
```
smetrics.nascar.com
nascar.com.ssl.sc.omtrdc.net
```

**Comment by yokoffing** [comment]
On Chrome with uBO Lite and Quad9 DNS, the videos won't load at all.

If I disable uBO Lite, the video loads.

**Comment by yokoffing** [comment]
Doesn't work with Firefox Strict ETP either. I have to disable both uBO and Strict ETP on Firefox.

**Comment by hagezi** [comment]
I can't get the videos to start at all. Regardless of whether I switch off browser tracking protection or not. I have tried Edge, Chrome, Thorium and Firefox. I have tried Cloudflare and Google DNS as upstream.

**Comment by xRuffKez** [comment]
Not loading, using unfiltered named.cache

**Comment by xenia-09A** [comment]
> @xenia-09A Thanks, I'll have to see if I can reproduce that in my region. Do you perhaps have the block page in NextDNS active?



> I can't get the videos to start at all. Regardless of whether I switch off browser tracking protection or not. I have tried Edge, Chrome, Thorium and Firefox. I have tried Cloudflare and Google DNS as upstream.
no that block page caused me many headaches it turned off for good ...
interesting.. I got the video to play just removing pro++, then putting it back broke it again

**Comment by hagezi** [comment]
Yes, unfortunately I can't find out for myself which domains need to be unblocked. I need someone to run the videos for that.

**Comment by xenia-09A** [comment]
I see smetrics.nascar.com being blocked unlblocking didn't help

**Comment by xRuffKez** [comment]
Fixed! You need to accept cookie consent!
#wontfix
@hagezi

**Comment by yokoffing** [comment]
related: https://www.reddit.com/r/uBlockOrigin/comments/1bv0s4q/

**Comment by hagezi** [comment]
Thanks @yokoffing, since you are getting the videos to run, could you test with the Pro++ to see if there is an additional blocklist issue?

**Comment by yokoffing** [comment]
@hagezi The videos need access to `moatads.com`.

I allowlisted `z.moatads.com` and `mb.moatads.com` in Control D, along with the adblock filter fix from Reddit, and the videos worked.

**Comment by hagezi** [comment]
Many thanks @yokoffing  for the great support. 

@xenia-09A This is definitely something for the personal allowlist if only one side is affected.

**Comment by xenia-09A** [comment]
> @hagezi The videos need access to `moatads.com`.
> 
> I allowlisted `z.moatads.com` and `mb.moatads.com` in Control D, along with the adblock filter fix from Reddit, and the videos worked. (Still broken with Firefox Strict ETP though.)

Hi on my side (in US) this domain is not queried at all 'moatads.com', to not in any logs when trying to watch the videos

**Comment by xenia-09A** [comment]
> Many thanks @yokoffing for the great support.
> 
> @xenia-09A This is definitely something for the personal allowlist if only one website is affected.

yes thank you I do now see z.moatads.com is blocked but this is a known tracker correct? but to keep my marriage happy I guess I can allow it , thanks so much LOL

**Comment by xenia-09A** [comment]
Hagezi do you agree with this?
Found in apple forum 

Moatads.com is certainly a distributor of malware... the link below is the report from threatcrowd.org which maps the various virus/malware/"corruption" the server transmits.

**Comment by hagezi** [comment]
No, its Advertising:

https://www.netify.ai/resources/domains/moatads.com

**Comment by xenia-09A** [comment]
> No, its Advertising/Tracking
> 
> https://www.netify.ai/resources/domains/moatads.com
> 
> https://duckduckgo.github.io/tracker-radar-wiki/domains/moatads.com.html

Thanks so much you efforts are a great help we really appreciate all these lists you maintain

**Comment by yokoffing** [comment]
Just a quick update @hagezi @xenia-09A: I disabled allowlisting `moatads.com` and used Hagezi's DNS - Pro list. I can watch videos on `nascar.com` with:

- [x] Brave with Aggressive Ads/Tracking protection
- [x] Chrome, even with adblocking and third-party cookies blocked
- [x] Firefox with Strict ETP, but thumbnails don't load (example: https://www.nascar.com/gallery/power-rankings-martinsville-to-texas-2024/).

**Comment by xenia-09A** [comment]
Thanks , it seems at one point they now required these trackers to load videos
